app/controllers/reports_controller.rb

```diff
@@ -103,11 +103,12 @@ def victims_list
     i = 0
     Victim.where(campaign_id: params[:id]).each do |victim|
       passwordSeen = Visit.where(:victim_id => victim.id).where('extra LIKE ?', "%password%").count > 0 ? "Yes" : "No"
+      fileDownload = Visit.where(:victim_id => victim.id).where('extra LIKE ?', "%ownload%").count > 0 ? "Yes" : "No"
```

Comment:
It's a hack for case sensitivity, so it matches Download and download.

If you'd be interested in this PR if it was more explicit I'll update.